regen.sh

```diff
@@ -42,18 +42,23 @@ if [ -z "$root" ]; then
   die "cannot find root of $PKG"
 fi
 
+remove_dirs=
+trap 'rm -rf $remove_dirs' EXIT
+
 if [ -z "$PROTOBUF" ]; then
-  protodir=$(mktemp -d -t regen-cds-proto.XXXXXX)
-  git clone -q $PROTO_REPO $protodir &
-  trap 'rm -rf $protodir' EXIT
+  proto_repo_dir=$(mktemp -d -t regen-cds-proto.XXXXXX)
+  git clone -q $PROTO_REPO $proto_repo_dir &
+  remove_dirs="$proto_repo_dir"
```

Comment:
Add the prefix here too, just to protect against mistakes reordering the file in future.

$remote_dirs $proto_repo_dir